x-pack/plugins/security_solution/public/common/components/navigation/breadcrumbs/index.ts

```diff
@@ -61,10 +61,16 @@ const isAdminRoutes = (spyState: RouteSpyState): spyState is AdministrationRoute
 
 // eslint-disable-next-line complexity
 export const getBreadcrumbsForRoute = (
-  object: RouteSpyState & TabNavigationProps,
+  objectParam: RouteSpyState & TabNavigationProps,
   getUrlForApp: GetUrlForApp
 ): ChromeBreadcrumb[] | null => {
-  const spyState: RouteSpyState = omit('navTabs', object);
+  const spyState: RouteSpyState = omit('navTabs', objectParam);
+
+  // Sets `timeline.isOpen` to false in the state to avoid reopening the timeline on breadcrumb click. https://github.com/elastic/kibana/issues/100322
+  const object = objectParam.timeline?.isOpen
```

Comment:
`const object = { ...objectParam, timeline: { ...objectParam.timeline, isOpen: false } }` is sufficient, right?

Reply:
Yes, always destructuring the object and setting `timeline.isOpen` would fix the bug. I was afraid that it might introduce new bugs when `objectParam.timeline` is undefined, but it is always defined. So I will follow your suggestion and simplify the code. Thank you!